lib/tasks/migrate.rake

```diff
@@ -261,6 +261,32 @@ namespace :migrate do
     end
   end
 
+  namespace :media do
+    # Media resources id 1 - 457799 are all subjects for project Snapshot Supernova.
+    # They currently exist outside of our media folder organizational structure.
+    # All files have already been copied over to /subject_location, this task must be run
+    # to update the media record src (which points to where the file is located)
+    desc 'Update src to subject_locations for Snapshot Supernova media'
+    task rewrite_supernova_file_locations: :environment do
+      # regex to check whether current src is an old path
+      OLD_PATH_REGEX = /\Apanoptes-uploads.zooniverse.org\/1\/0\//.freeze
+      # this will return match data that will give us just the file name,
+      # which is the part of the string following the old path
+      CAPTURE_GROUP_REGEX = /\A(?:panoptes-uploads.zooniverse.org\/1\/0\/)?(.+)\z/.freeze
+
+      puts 'starting supernova src location rewrite'
+      Media.where('id < 457800').find_each do |medium|
```

Comment:
super minor - OOI is 457800 a supernova project subject? If so the `where` clause should match it as well `'id <= ?'`
```suggestion
      last_known_supernova_project_subject_id = 457800
      Media.where('id < ?', last_known_supernova_project_subject_id').find_each do |medium|
```

Reply:
Nope it is not. I'll make it more explicit with `id <= 457799`